.github/workflows/all.yml

```
@@ -237,13 +238,13 @@ jobs:
 
           # Log into GitHub Container Registry
           echo "${{ secrets.CR_PAT }}" | docker login https://ghcr.io -u ${{ github.actor }} --password-stdin
-          docker tag "$DOCKER_IMAGE_ID" "ghcr.io/$DOCKER_IMAGE_ID:$VERSION"
-          docker push "ghcr.io/$DOCKER_IMAGE_ID:$VERSION"
+          docker tag "$DOCKER_IMAGE_ID" "ghcr.io/$GHCR_IMAGE_ID:$VERSION"
```

Shouldn't this be:

```suggestion
          docker tag "$GHCR_IMAGE_ID" "ghcr.io/$GHCR_IMAGE_ID:$VERSION"
```

?

That is, we want the GHCR image to be accessible via `k6io/k6` and the Docker Hub one via `loadimpact/k6` (until we move the Docker Hub org as well).